build/includes/terraform.mk

```diff
@@ -80,10 +79,7 @@ endif
 		-var node_count=$(GCP_CLUSTER_NODEPOOL_INITIALNODECOUNT)'
 	GCP_CLUSTER_NAME=$(GCP_TF_CLUSTER_NAME) $(MAKE) gcloud-auth-cluster
 
+gcloud-terraform-destroy-cluster: GCP_PROJECT ?= $(current_project)
 gcloud-terraform-destroy-cluster:
-ifndef GCP_PROJECT
-	$(eval GCP_PROJECT=$(shell sh -c "gcloud config get-value project 2> /dev/null"))
-endif
-	$(DOCKER_RUN) bash -c 'cd $(mount_path)/build/terraform/gke && \
-	terraform destroy -var project=$(GCP_PROJECT) -target module.helm_agones.helm_release.agones -auto-approve && sleep 60 && \
-	terraform destroy -var project=$(GCP_PROJECT) -auto-approve'
+	$(DOCKER_RUN) bash -c 'cd $(mount_path)/build/terraform/gke && terraform destroy -var project=$(GCP_PROJECT) -auto-approve'
```

Comment:
Do we no longer need the sleep and re-run?

Reply:
The helm3 terraform module solves the issue with cleaning up helm resources before deleting the cluster! 👍